web_common/web_common/styles/main.scss

```diff
@@ -171,6 +171,11 @@ a {
     left: 36px;
 }
 
+.benchmark-caret:after {
+    /* 74 (measured width of Notebook) / 2 + 9 - 20 / 2 = 38px */
```

Comment:
Can you change the comment here to say how you computed this number? I.e. what is the width of Benchmark?

Reply:
the number is wrong, didn't know how to measure the width so I put it as a placeholder

Reply:
I'm going to assume that Cotton inspected the HTML element and got the width of the text "Benchmark" in the header in dev deploy and then used that width.

Reply:
I tried to dev deploy your branch to figure out what the correct width here should be, but you need to rebase it on master first. You can choose to skip adding the benchmark to the header (save it for a different PR). If you decide to do that, then let's remove it from the file below `header.html`.